proto/api_spec/method.proto

```diff
@@ -190,6 +190,13 @@ message HTTPBody {
 
   // Preserve original value if "other"
   string other_type = 2;
+
+  enum Errors {
+    UNKNOWN_ERROR = 0;
+    PARSING_ERROR = 1;
```

Comment:
I initially wrote "I understand we can't use 0 as no error" because it's the default value in existing witnesses.

But, would it be so bad as to treat all existing witnesses as "no error" because we don't want to treat them specially?

My feedback here is that we should try to distinguish between 'no error' and 'unknown error'.  Maybe that's what you intended and the name is just not working for me.